experimental/Experimental.jl

```diff
@@ -18,9 +18,9 @@ const oldexppkgs = [
 # For more background, see https://github.com/oscar-system/Oscar.jl/issues/2300.
 const orderedpkgs = [
   "LieAlgebras",
-  "BasisLieHighestWeight",   # nees code from LieAlgebras
+  "BasisLieHighestWeight",   # needs code from LieAlgebras
 ]
-exppkgs = filter(x->isdir(joinpath(expdir, x)) && !(x in oldexppkgs) && !(x in orderedpkgs), readdir(expdir))
+exppkgs = filter(x->isdir(joinpath(expdir, x)) && !(x in oldexppkgs) && !(x in orderedpkgs) && (x != "FTheoryTools"), readdir(expdir))
```

Comment:
This looks suspicious, why is this necessary? If this needs a specific ordering it can be added to the list above?

```suggestion
exppkgs = filter(x->isdir(joinpath(expdir, x)) && !(x in oldexppkgs) && !(x in orderedpkgs), readdir(expdir))
```

Reply:
I need `FTheoryTools` to be loaded after the experimental `schemes` code... and found the above solution.

Key: The `schemes` code has not been formatted according to the "new experimental packages style" (i.e. with its docs, tests etc. in the experimental folder), whereas `FTheoryTools` complies with this new standard.

Hence, we are mixing new-style and old-style experimental packages and expect them to be loaded in a certain order (here: Schemes first, then FTheoryTools).

However, as I see it, it is currently hard coded that the old-style experimental packages are loaded after the new-style experimental packages. And so, it seemed that my code could only work with some additional tweaking: 
* The line that you highlight excludes excludes `FTheoryTools` to be considered as an old-style experimental package. (Otherwise, there is an error, as the code cannot find the include files for `FTheoryTools` under the assumption that it is an old-style package).
* `FTheoryTools` has been removed from the list of the new-style experimental packages. (Otherwise, it would be loaded before schemes, which leads to `undefvar errors`, as we need schemes functionality now.)
* Finally, the very last line of the `Experimental.jl` file includes/loads the `FTheoryTools`.

Not pretty, and am happy to improve. For now, it seemed to work though.